app/controllers/experiments_controller.rb

```diff
@@ -535,6 +535,10 @@ def experiment_moes
     render json: moes_info
   end
 
+  def experiment_results
```

Comment:
Coś w tym jest, ale myślę, że experiment_results_and_error_reason nie jest dobrą nazwą :) już wolę to co jest. Chyba, że ktoś wymyśli coś innego.